core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -495,7 +497,8 @@ private List<FlowWithTriggers> computeSchedulable(List<FlowWithSource> flows, Li
     public List<FlowWithTriggers> schedulerTriggers() {
         Map<String, FlowWithSource> flows = this.flowListeners.flows()
             .stream()
-            .collect(Collectors.toMap(FlowWithSource::uidWithoutRevision, Function.identity()));
+            .map(flow -> pluginDefaultService.injectAllDefaults(flow, false))
```

Comment:
Don't we already have the defaults as they are injected inside the flow listener consumer?